### PR TITLE
each element of Xi array in Fluid.Examples.TraceSubstances.RoomCO2WithControls

### PR DESCRIPTION
**Pull request by thorade**
also see #2352 

https://libraries.openmodelica.org/branches/master/Modelica_trunk/files/Modelica_trunk_Modelica.Fluid.Examples.TraceSubstances.RoomCO2WithControls.err

### CODE REVIEW

File: Modelica/Fluid/Examples/TraceSubstances.mo [hunk 1]
```diff
@@ -251,4 +251,5 @@ of magnitude.
 </p>
 </html>"));
   end RoomCO2WithControls;
+
```
Comment (beutlich) [inline Modelica/Fluid/Examples/TraceSubstances.mo]: Not necessary for the pull request.

**Comment by thorade** [comment]
Done.

**Comment by thorade** [comment]
Should line 106 be `each Xi(each)`?

**Comment by beutlich** [comment]
> Should line 106 be each Xi(each)?

Not sure. I leave that one open therefore.

**Comment by HansOlsson** [comment]
> Should line 106 be  each Xi(each) ?

Yes as far as I can see, (well, `each Xi(each nominal=0.01)` )- since Xi is an array inside an array of components (mediums).

**Pull request merged**